core/test/matrix/ell.cpp

```diff
@@ -231,3 +231,49 @@ TYPED_TEST(Ell, CanBeReadFromMatrixAssemblyData)
 
     this->assert_equal_to_original_mtx(m);
 }
+
+
+TYPED_TEST(Ell, RecognizesInfiniteValue)
+{
+    using value_type = typename TestFixture::value_type;
+    using index_type = typename TestFixture::index_type;
+    value_type values[] = {INFINITY, 3.0, 4.0, -1.0, 2.0, 0.0, 0.0, -1.0};
+    index_type col_idxs[] = {0, 1, 0, -1, 1, 0, 0, -1};
+
+    auto mtx = gko::matrix::Ell<value_type, index_type>::create(
```

Comment:
```suggestion
    index_type col_idxs[] = {0, 1, 0, -1, 1, 0, 0, -1};
    auto mtx = gko::matrix::Ell<value_type, index_type>::create(
```

Because your action and assert are in the same line, the assign part should not have a space.
you can use comment to distinguish if you want.

Reply:
It is not resolved yet